code/modules/food&drinks/drinks/drinks.dm

```diff
@@ -152,6 +152,16 @@
 	item_state = "coffee"
 	list_reagents = list("hot_coco" = 30, "sugar" = 5)
 
+/obj/item/weapon/reagent_containers/food/drinks/weightloss
+	name = "Weight-Loss Shake"
+	desc = "A shake designed to cause weight loss.  The package proudly proclaims that it is 'tapeworm free.'"
+	icon_state = "coffee"
+	item_state = "coffee"
+	New()
+		..()
+		reagents.add_reagent("lipolicide", 30)
+		reagents.add_reagent("chocolate", 5)
+
```

Comment:
Adding a dangerous poison (I'm pretty sure this stuff is in teh traitor poison bundle) to the drinks vendors seems like a bad idea, especially at 30u (!) in each one.